db/seeds.rb

```diff
@@ -8,7 +16,7 @@
       email_address: hash.fetch('email', nil),
       enabled: hash.fetch('enabled', true),
       estate: hash.fetch('estate'),
-      id: hash.fetch('id'),
+      id: prison_uuid_mappings[path],
```

Comment:
if the mapping isn't found, does it attempt to create a null `id` and fail, or does the DB-generated UUID take precedence?